src/vsr/replica.zig

```diff
@@ -6915,7 +6947,7 @@ pub fn ReplicaType(
         ///
```

Comment:
The comment for `fn op_prepare_max` up above needs to be updated, since right now it says

> Receiving and storing an op higher than `op_prepare_max()` is forbidden;